scene/gui/file_dialog.cpp

```diff
@@ -178,8 +178,16 @@ void FileDialog::_post_popup() {
 	set_process_unhandled_input(true);
 
 	// For open dir mode, deselect all items on file dialog open.
-	if (mode == MODE_OPEN_DIR)
+	if (mode == MODE_OPEN_DIR) {
 		deselect_items();
+		file_label->set_visible(false);
```

Comment:
Instead of hiding all three controls manually, maybe we can just hide the `hbc` that contains them? (And then `file_label` no longer needs to be a member variable).

Reply:
`hbc` would have to become a member variable on the other hand.

Reply:
Something like this: https://hastebin.com/keyujejovu.diff
(+ similar changes to `EditorFileDialog`)